src/Reflection/ReflectionClass.php

```diff
@@ -68,16 +68,16 @@ class ReflectionClass implements Reflection
     /** @var ClassLikeNode */
     private $node;
 
-    /** @var ReflectionClassConstant[]|null indexed by name, when present */
+    /** @var array<string, ReflectionClassConstant>|null indexed by name, when present */
     private $cachedReflectionConstants;
 
-    /** @var ReflectionProperty[]|null */
+    /** @var array<string, ReflectionProperty>|null */
     private $cachedImmediateProperties;
 
     /** @var ReflectionProperty[]|null */
```

Comment:
What's the type of this array's keys?

Reply:
I added it. I couldn't deduce it at first so I skipped it.